packages/postman-to-openapi/src/convert.test.ts

```diff
@@ -0,0 +1,247 @@
+import { beforeAll, describe, expect, it } from 'vitest'
+
+import { downloadFileToMemory } from '../../openapi-parser/tests/utils/downloadFileGcp'
```

Comment:
No relative sibling package imports. For this particular use case you can just use a fetch function with the URL to the bucket.